enhancements/cluster-logging/cluster-logging-otel-support.md

```diff
@@ -0,0 +1,294 @@
+          - host.name
+          - process.command
+          # For openshift log source auditd
+          - node.name
+          # For openshift log sources kubeAPI and openshiftAPI
+          - url.path
+          - http.request.method_original
+          - http.response.status.code
+```
+
+#### Log Visualization
+
+The **log-view-plugin** will be modified to replace its UI widgets to rely upon OTEL attributes in lieu of ViaQ.  (Review meeting notes for additional info)
+
+#### Log Collection and Forwarding
+
+* The **cluster-logging-operator** will be updated to evaluate the OTLP annotation:
+    * Absence of the annotation implies forwarding to **LokiStacks** using ViaQ data model
+    * Existance of the annotation, the operator:
+        * will validate the list of names maps to a defined, **LokiStack** output and rejects the **ClusterLogForwarder** for any mismatches
+        * will internally initialize OTLP outputs for the enabled **LokiStacks**, in lieu of Loki outputs
+        * will configure **LokiStack** OTLP outputs to include MVP labels
+
+Once this enhancement achieves General Acceptance, OTEL will be the default data model when forwarding logs to **LokiStack**.  Cluster administrators will
+be able to support the Tech Preview behavior until it is removed by annotating the **ClusterLogForwarder** like the following:
+
+```
+   logstore.observability.openshift.io/viaq: [[LOKISTACK_NAMES]]
+```
+* LOKISTACK_NAMES := Comma delimited list of **clusterlogforwarder.spec.output[].name** of type **lokiStack**
+
+
+* The **cluster-logging-operator** will be updated to evaluate the ViaQ annotation:
+    * Absence of the annotation implies forwarding to **LokiStacks** using OTEL data model
+    * Existance of the annotation, the operator:
+        * will validate the list of names maps to a defined, **LokiStack** output and rejects the **ClusterLogForwarder** for any mismatches
+        * will internally initialize Loki outputs for the enabled **LokiStacks**, in lieu of OTLP outputs
+        * will configure **LokiStack** LokiStack outputs to include OTEL equivalent MVP labels
+
+52
+### Risks and Mitigations
+
+### Drawbacks
+
+This design offers a transitional plan to allow users to change the enrichment model of their log storage while contining to use the console web UI plugin and
+MVP attributes.  Advanced queries using [logQL](https://grafana.com/docs/loki/latest/query/) that rely upon attributes other then MVP may no longer function.
+
+
+## Open Questions [optional]
+
+1. Do we need to forward the complimentry UI OTEL attributes when configured for ViaQ?
+
+## Test Plan
+
+## Graduation Criteria
+
+### Dev Preview -> Tech Preview
+
+- Ability to conditionally configure the datamodel(ViaQ or OTEL) when forwarding to in-cluster LokiStack
+- Ability to retrieve OTEL formatted log entries from in-cluster LokiStack using ViaQ attributes: log_type, kubernetes_namespace_name, kubernetes_container_name, kubernetes_pod_name
```

Comment:
Our current set of labels/attributes is [this](https://github.com/openshift/cluster-logging-operator/blob/master/internal/generator/vector/output/loki/loki.go#L34-L51):

- `log_type`
- `kubernetes.container_name`
- `kubernetes.host`
- `kubernetes.namespace_name`
- `kubernetes.pod_name`

I think these are good candidates for the "required" set, maybe we should add the new OTLP counterparts to this list as well.

Reply:
adding `log_source` to the list.

Are you suggesting we alter this text to identify both sets side-by-side?

Reply:
Resolving as document was modified to move these to a dedicated document